sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -126,6 +127,16 @@ def key_id(self):
         """
         return self._key_id.source_id
 
+    @classmethod
+    def from_jwk(cls, jwk):
+        # type: (dict) -> CryptographyClient
+        """Creates a client that can only perform cryptographic operations locally.
+
+        :param dict jwk: the key's cryptographic material, as a dictionary.
+        """
+        key_id = "https://key-vault.vault.azure.net/keys/local-key"
+        return cls(KeyVaultKey(key_id, jwk), object, _local_only=True)
```

Comment:
Is there a reason to pass the type `object` rather than an instance of it?

Reply:
None in particular -- probably makes more sense to pass an instance